src/index.html

```diff
@@ -11,7 +11,190 @@
+        </div>
+
+        <div class="form-field">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="terms"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              minlength="5"
+              maxlength="15"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input
+              type="radio"
+              value="yes"
+              name="cats"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            value="yes"
+            name="cats"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="time"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What are your favorite brand cars?
+            <select
+              name="cars"
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+              <option value="blank"></option>
+            </select>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              name="rate"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <div class="form-field">
+          <label>
+            Comments:
+            <textarea
+              name="comments"
+              autocomplete="off"
+           ></textarea>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Would you recommend us?
+            <select
+              name="recommendation"
+            >
```

Comment:
you can write it in one line
```suggestion
            <select name="recommendation">
```